FatFs_SPI/sd_driver/sd_card.c

```diff
@@ -450,15 +453,21 @@ static const char *cmd2str(const cmdSupported cmd) {
 #define SD_COMMAND_TIMEOUT 2000 /*!< Timeout in ms for response */
 
 static int sd_cmd(sd_card_t *pSD, const cmdSupported cmd, uint32_t arg,
-                  bool isAcmd, uint32_t *resp) {
+                  bool isAcmd, uint32_t *resp, absolute_time_t *p_timeout_time) {
     TRACE_PRINTF("%s(%s(0x%08lx)): ", __FUNCTION__, cmd2str(cmd), arg);
 
     int32_t status = SD_BLOCK_DEVICE_ERROR_NONE;
     uint32_t response;
+    absolute_time_t default_timeout_time = make_timeout_time_ms(SD_COMMAND_TIMEOUT);
+
+    if (p_timeout_time == NULL)
+    {
```

Comment:
Please use K&R formatting, for consistency with the rest of the library.
(I use
    ` "C_Cpp.clang_format_style": "{ BasedOnStyle: Google, IndentWidth: 4, ColumnLimit: 0}",`
)